app/utils/auth.ts

```diff
@@ -53,6 +55,17 @@ export const formatRoleName = (role: string): string => {
   }
 };
 
+/**
+ * Return the text from an email up to the first `.`
+ *
+ * @example
+ * // returns "John"
+ * formatFirstNameFromEmail("john.c@austintexas.gov")
+ */
+export const formatFirstNameFromEmail = (email: string): string => {
+  return email?.split(".")[0] || "";
```

Comment:
:sparkles:

Reply:
This is pretty cool, but I fear it's brittle. If this returns the empty string, this whole thing gets wonky on the page with a dangling `,!`. I think your comment below about the greeting thing being cutesy is the real answer, and I'd strike the whole salutation entirely. Just my two cents there.